core/test/validation_test.jl

```diff
@@ -438,6 +438,70 @@ end
     )
 end
 
+@testitem "tabulated rating curve validation" begin
+    import SQLite
+    using Ribasim: valid_tabulated_curve_level
+    using Logging
+
+    toml_path =
+        normpath(@__DIR__, "../../generated_testmodels/level_range/ribasim.toml")
+    @test ispath(toml_path)
+    invalid_level = -0.1
+
+    config = Ribasim.Config(toml_path)
+    model = Ribasim.Model(config)
+
+    db_path = Ribasim.input_path(config, config.database)
+    db = SQLite.DB(db_path)
+    parameters = Ribasim.Parameters(db, config)
```

Comment:
It looks like this is not needed, the parameters are already in the model and can be retrieved as `model.integrator.p`. Same below.